tests/foreman/api/test_multiple_paths.py

```diff
@@ -632,40 +632,37 @@ class EntityReadTestCase(APITestCase):
     function correctly.
     """
 
-    # Most entities are commented-out because they do not inherit from
-    # EntityReadMixin, due to issues with data returned from the API.
-    #
     # ComputeResource entities cannot be reliably read because, depending upon
     # the provider, different sets of attributes are returned. For example, the
     # "uuid" attribute is only returned for certain providers. Perhaps multiple
     # types of compute resources should be created? For example:
     # LibvirtComputeResource.
     @data(
-        # entities.ActivationKey,
+        entities.ActivationKey,
```

Comment:
More test cases! Yay!

No, really. The benefit to running these tests is that we can better detect when bugs like [this](https://bugzilla.redhat.com/show_bug.cgi?id=1193149) crop up.